blocks/core/ff_module/ff_module-task/ff_module-task.xml

```diff
@@ -1,4 +1,4 @@
-<notice id="{{ id }}" display_date_type="{{ display_date_type }}" sent_to="{{ sent_to }}" no_excused="{{ no_excused }}" handedin="{{ completed_by }}" marked="{{ marked }}" type="onlinetask" from="{{ from }}" from_href="{{ from_href }}" link_href="{{ link_href }}" to="{{ to }}" duedate="{{ duedate }}" startdate="{{ startdate }}" setdate="{{ setdate }}">
+<notice modifier="{{ modifier }}" id="{{ id }}" sent_to="{{ sent_to }}" no_excused="{{ no_excused }}" handedin="{{ completed_by }}" marked="{{ marked }}" from="{{ from }}" link_href="{{ link_href }}" to="{{ to }}" duedate="{{ duedate }}">
```

Comment:
They weren't being used anywhere in the xsl, so made what was being used less clear.